libvast/src/format/mrt.cpp

```diff
@@ -87,12 +120,11 @@ expected<event> reader::read() {
   mrt::record r;
   if (!parser_(buffer_, r))
     return make_error(ec::format_error, "failed to parse MRT message");
+  if(r.header.subtype == 1)
+    VAST_DEBUG(this->name(), "type", r.header.type, "subtype", r.header.subtype);
```

Comment:
I'd keep your own debug comments that should not be exposed locally. You can add subsets of your changeset with `git add -p`.